include/seastar/net/socket_defs.hh

```diff
@@ -123,6 +133,14 @@ struct ipv6_addr {
     }
 };
 
+struct ud_addr {
+    std::filesystem::path sfile_;
+
+    explicit ud_addr(const std::string&);
+    explicit ud_addr(std::string_view fn) { sfile_ = fn; }
```

Comment:
this requires C++17. use `seastar::compat::string_view` instead. or just drop this construct to be consistent with its cousins.

Reply:
Removed